tests/large/test_ide.py

```diff
@@ -450,3 +450,50 @@ def test_default_install(self):
         self.expect_and_no_warn("Arduino is already installed.*\[.*\] ")
         self.child.sendline()
         self.wait_and_close()
+
+
+class BaseNetBeansTests(LargeFrameworkTests):
+    """The base for all Netbeans installers."""
+
+    TIMEOUT_INSTALL_PROGRESS = 600
+    TIMEOUT_START = 60
+    TIMEOUT_STOP = 60
+
+    def setUp(self):
+
+        super().setUp()
+        self.installed_path = os.path.expanduser("~/tools/ide/netbeans")
+        self.desktop_filename = "netbeans.desktop"
+
+    @property
+    def arch_option(self):
```

Comment:
You don't use that property, does it means that 32 and 64 bits are the same?

If so, please remove that property to avoid dead test code.